documentation/snippet_embed_template.html

```diff
@@ -0,0 +1,27 @@
+<div id="container" style="min-width: 600px; min-height: 300px"></div>
```

Comment:
You'll want to generate a proper html file, e.g. start with

```
<!DOCTYPE html>
<html>
```

use `<head>` for the scripts and `body` for the content, etc